src/simcore_service_deployment_agent/auto_deploy_task.py

```diff
@@ -194,6 +198,10 @@ async def deploy_stacks(
             url, app_session, bearer_code, config["stack_name"]
         )
         if not current_stack_id:
+            log.warning(
+                "During polling: Stack does not exist, did it vanish or not initialize correctly? Will create new stack."
```

Comment:
not sure I understand this warning. "During polling:" is this something that happens in another case?

Reply:
adressed